libbeat/autodiscover/providers/kubernetes/config.go

```diff
@@ -30,11 +30,16 @@ import (
 // Config for kubernetes autodiscover provider
 type Config struct {
 	KubeConfig     string        `config:"kube_config"`
-	Host           string        `config:"host"`
 	Namespace      string        `config:"namespace"`
 	SyncPeriod     time.Duration `config:"sync_period"`
 	CleanupTimeout time.Duration `config:"cleanup_timeout" validate:"positive"`
 
+	// Needed when resource is a pod
+	Host string `config:"host"`
+	// Scope can be either host or cluster.
+	Scope    string `config:"scope"`
+	Resource string `config:"resource"`
```

Comment:
both scope and resource are new, could you update the docs?

Reply:
updated docs.